documentation/SA1400.md

```diff
@@ -25,6 +25,8 @@ C# allows elements to be defined without an access modifier. Depending upon the
 
 This rule requires an access modifier to be explicitly defined for every element. This removes the need for the reader to make assumptions about the code, improving the readability of the code.
 
+Members declared in an `interface` are implicitly `public` and do not need an explicit access modifier. This includes default interface members introduced in C# 8 (instance members with default implementations, static interface members, and nested interface members such as delegates). SA1400 does not report diagnostics for these interface members.
```

Comment:
I agree with this point.
I wonder if we should have a rule to discourage adding redundant `public` access modifiers on interface members when they are all `public`. I've seen some of these crop up already, and IMO it is more confusing and maybe mis-educating other developers who read it into thinking it's important.

Reply:
➡️ Would be good to file this as a separate request. It may already exist in the IDE/SDK code style rules.